ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -1667,6 +1667,20 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-ovn-ipsec
+- as: azure-ipi-ovn-ipsec-p3-f28-cloud
```

Comment:
@sunzhaohua2 by a chance, I saw this testing is really similar with the following `azure-ipi-p3-f28-long-duration-part3-3`, both of them are running `openshift-e2e-test-qe-long-duration`, can we merge them into one?

Reply:
`azure-ipi-p3-f28-long-duration-part3-3`  is `Cluster_Operator` team's

Reply:
Yeah, I see. But they are running the same test chain, and also the same `E2E_RUN_TAGS`

Reply:
Got it, yes we can

Reply:
cc @jianping-shu to involve the discussion.

cc @liangxia do you know something about the limit to the job running time?

Reply:
and seem like we need to combine `Cluster_Operator` and `Cluster_Infrastructure` together for `TEST_SCENARIOS`.

Reply:
I'm fine to combine tests. The hard limit is 24 hours ( can not overwrite) in Prow, and another "soft" limit which can be defined by each step (can be updated, but no more than 24 hours)

Reply:
Do you know what is the format of `TEST_SCENARIOS` setting to cover both `Cluster_Operator` and `Cluster_Infrastructure`?

Reply:
Oh, seem like it is `Cluster_Operator|Cluster_Infrastructure`

Reply:
@sunzhaohua2 combined now. If any more update, pls let me know.

Reply:
@jianlinliu As I know, the long duration jobs are splitted to several parts because of the 24 hours time limit concern. The Cluster_Operator job (part3-3) currently takes 5-6 hours, and Cluster_Infrastructure job takes 7h, so I suggest to keep them separate since we are keeping to add many more Hive cases into the job, thanks

Reply:
okay. Move back the original job. cc @sunzhaohua2

Reply:
Thanks!